src/Helper/Mapping.php

```diff
@@ -56,91 +56,360 @@ public static function delivery_type() {
+							'combination' => array(),
+							'code'        => '4907',
+							'options'     => array(
+								array(
+									'characteristic' => '005',
+									'option'         => '025',
+								),
+								array(
+									'characteristic' => '101',
+									'option'         => '012',
+								)
+							)
+						)
 					),
 				),
 				'ROW' => array(
-					'delivery_day'  => array(
-						'4945' => array(),
-					),
+					'delivery_day'  => self::globalpack_products(),
 					'pickup_points' => array(
-						'4945' => array(),
+						array(
+							'combination' => array(),
+							'code'        => '4909',
+							'options'     => array(
+								array(
+									'characteristic' => '005',
+									'option'         => '025',
+								)
+							)
+						)
 					),
-				),
+				)
 			),
 			'BE' => array(
-				'BE' => array(
+				'BE'  => array(
 					'delivery_day'  => array(
-						'4961' => array(),
-						'4960' => array( 'only_home_address' ),
-						'4963' => array( 'signature_on_delivery' ),
-						'4962' => array( 'signature_on_delivery', 'only_home_address' ),
-						'4965' => array( 'insured_shipping', 'only_home_address' ),
+						array(
+							'combination' => array(),
+							'code'        => '4961',
+							'options'     => array()
+						),
+						array(
+							'combination' => array( 'only_home_address' ),
+							'code'        => '4960',
+							'options'     => array()
+						),
+						array(
+							'combination' => array( 'signature_on_delivery' ),
+							'code'        => '4963',
+							'options'     => array()
+						),
+						array(
+							'combination' => array( 'signature_on_delivery', 'only_home_address' ),
+							'code'        => '4962',
+							'options'     => array()
+						),
+						array(
+							'combination' => array( 'insured_shipping', 'only_home_address' ),
+							'code'        => '4965',
+							'options'     => array()
+						)
 					),
 					'pickup_points' => array(
-						'4880' => array(),
-						'4878' => array( 'insured_shipping' ),
+						array(
+							'combination' => array(),
+							'code'        => '4880',
+							'options'     => array()
+						),
+						array(
+							'combination' => array( 'insured_shipping' ),
+							'code'        => '4878',
+							'options'     => array()
+						)
 					),
 				),
-				'NL' => array(
+				'NL'  => array(
 					'delivery_day' => array(
-						'4944' => array(),
+						array(
+							'combination' => array(),
+							'code'        => '4944',
+							'options'     => array()
+						)
 					),
 				),
-				'EU' => array(
-					'delivery_day' => array(
-						'4944' => array(),
-					),
+				'EU'  => array(
+					'delivery_day' => self::european_shipment_products(),
 				),
 				'ROW' => array(
-					'delivery_day' => array(
-						'4945' => array(),
-					),
+					'delivery_day' => self::globalpack_products()
+				)
+			)
+		);
+	}
+
+	/**
+	 * Products code & options available for European and GlobalPack Shipments.
+	 *
+	 * @return array[]
```

Comment:
always add period at the end of comment line.
```suggestion
	 * @return array[].
```